bgc_data/bgc_larval_fish_map.sql

```diff
@@ -8,11 +8,11 @@ CREATE MATERIALIZED VIEW bgc_larval_fish_map AS
       lfs.longitude AS "Longitude",
       lfs.trip_code AS "TripCode",
       lfs.sampledateutc AS "SampleTime_UTC",
-      to_char(lfs.sampledatelocal, 'YYYY-MM-DD HH24:MI:SS') AS "SampleTime_local",
-      extract(year from lfs.sampledatelocal)::int AS "Year_local",
-      extract(month from lfs.sampledatelocal)::int AS "Month_local",
-      extract(day from lfs.sampledatelocal)::int AS "Day_local",
-      to_char(lfs.sampledatelocal, 'HH24:MI') AS "Time_local24hr",
+      to_char(lfs.sampledatelocal, 'YYYY-MM-DD HH24:MI:SS') AS "SampleTime_Local",
```

Comment:
This one is configured as a [filter in geoserver-config](https://github.com/aodn/geoserver-config/blob/4f2fcd96011efe51b44d809c70344758e9247364/workspaces/imos/JNDI_imos_bgc_db/bgc_larval_fish_count_raw_data/filters.xml#L18), so that will need updating too.
I think we switched all the other filters to UTC, but please double-check.

Reply:
Updated [here](https://github.com/aodn/geoserver-config/pull/739/commits/964ba07b300d93efd161beef6246ad01d25923a9). I checked and seems that larval fish is the only instance where we are using local time to filter, everywhere else we use UTC.